modules/installation-vsphere-installer-infra-requirements.adoc

```diff
@@ -40,6 +40,19 @@ A user requires the following privileges to install an {product-title} cluster:
 
 For more information about creating an account with only the required privileges, see link:https://docs.vmware.com/en/VMware-vSphere/7.0/com.vmware.vsphere.security.doc/GUID-5372F580-5C23-4E9C-8A4E-EF1B4DD9033E.html[vSphere Permissions and User Management Tasks] in the vSphere documentation.
 
+[discrete]
+[id="installation-vsphere-installer-infra-requirements-vmotion_{context}"]
+== Using {product-title} with vMotion
```

Comment:
@sferich888 or @reestr Here we are updating our description of what we support for vMotion (compute) and what we don't support (Storage vMotion). PTAL and ack if this looks okay to you.

Reply:
"It sounds to me that we should keep the "unsupported" language and further explain that this is because the use of Storage vMotion can cause issues." do we want to say that here?

Reply:
Yes, I have incorporated that wording into the `IMPORTANT` admonition.